dipy/viz/tests/test_actors.py

```diff
@@ -26,10 +26,13 @@
 if actor.have_vtk:
     if actor.major_version == 5 and use_xvfb:
         skip_slicer = True
+        skip_surface = True
     else:
         skip_slicer = False
+        skip_surface = False
```

Comment:
Using the option `offscreen=True` with `window.snapshot`, there is no reason to skip this test.

Reply:
fixed